frontend/src/components/eMIB/ActionViewEmail.jsx

```diff
@@ -99,6 +99,9 @@ class ActionViewEmail extends Component {
   // the return is a string in the following format:
   //  "<name 1> (<role 1>), <name 2> (<role 2>), ...""
   generateEmailNameList(contactIdList) {
+    if (contactIdList === undefined) {
```

Comment:
Are you sure that you condition here is working properly? When I used the _undefined_ conditions, I had to do as follow:
`if (typeof contactIdList === "undefined")`

Maybe it's working, but that is definitely something to take a look at.

Reply:
I know that without this if, it breaks on line 106 `for (let id of contactIdList) {` because contactIdList is undefined. It breaks on that line on master, but is now caught here (one of my tests also breaks here on master; I tried it before commiting). I'll look to see which is the "better" way to check

Reply:
As discussed offline, these appear to be interchangeable: https://stackoverflow.com/a/40735876